Analysis/include/SampleDescriptor.h

```diff
@@ -20,15 +20,14 @@ struct AnalyzerSetup {
     Period period;
     SignalMode mode;
     QCDmethod qcd_method;
+    std::vector<std::string> qcd_shape;
     DiscriminatorWP tauID_wp;
-    std::vector<std::string> final_variables;
     std::vector<double> pt_sel_bins;
-    bool apply_mass_cut{false}, apply_os_cut{true}, apply_iso_cut{true};
-    bool use_kinFit{false}, use_svFit{false}, applyTauID{true};
+    bool use_kinFit{false}, use_svFit{false};
     std::set<UncertaintySource> unc_sources;
     EventCategorySet categories;
     EventSubCategorySet sub_categories;
-    EventRegionSet regions;
+     std::vector<std::string> regions;
```

Comment:
Don't change regions type - it will result in modifications in various places in the code, which is better to avoid.
```
EventRegionSet regions;
std::vector<std::string> regions_str;
```